notebooks/115-async-api/115-async-api.ipynb

```diff
@@ -324,28 +320,32 @@
     "        # Start capturing\n",
     "        start_time = time.time()\n",
     "        player.start()\n",
-    "        # Read CURRENT frame\n",
-    "        frame = player.next()\n",
     "        if use_popup:\n",
     "            title = \"Press ESC to Exit\"\n",
     "            cv2.namedWindow(title, cv2.WINDOW_GUI_NORMAL | cv2.WINDOW_AUTOSIZE)\n",
+    "        # Capture CURRENT frame\n",
+    "        frame = player.next()\n",
+    "        resized_frame = preprocess(frame)\n",
+    "        curr_request.set_tensor(input_layer_ir, ov.Tensor(resized_frame))\n",
+    "        # Start the CURRENT inference request\n",
+    "        curr_request.start_async()\n",
     "        while True:\n",
-    "            # Read NEXT frame\n",
+    "            # Capture NEXT frame\n",
     "            next_frame = player.next()\n",
-    "            if frame is None:\n",
+    "            if next_frame is None:\n",
     "                print(\"Source ended\")\n",
     "                break\n",
     "            # if frame larger than full HD, reduce size to improve the performance\n",
-    "            scale = 1280 / max(frame.shape)\n",
+    "            scale = 1280 / max(next_frame.shape)\n",
     "            if scale < 1:\n",
-    "                frame = cv2.resize(frame, None, fx=scale, fy=scale, interpolation=cv2.INTER_AREA)\n",
-    "            resized_frame = preprocess(frame)\n",
-    "            curr_request.set_tensor(input_layer_ir, ov.Tensor(resized_frame))\n",
-    "            # Start the CURRENT inference request\n",
-    "            curr_request.start_async()\n",
-    "            # Waiting for NEXT inference result\n",
-    "            if next_request.wait_for(-1) == 1:\n",
-    "                res = next_request.get_output_tensor(0).data\n",
+    "                next_frame = cv2.resize(next_frame, None, fx=scale, fy=scale, interpolation=cv2.INTER_AREA)\n",
+    "            resized_frame = preprocess(next_frame)\n",
+    "            next_request.set_tensor(input_layer_ir, ov.Tensor(resized_frame))\n",
+    "            # Start the NEXT inference request\n",
+    "            next_request.start_async()\n",
+    "            # Waiting for CURRENT inference result\n",
+    "            if curr_request.wait_for(-1) == 1:\n",
+    "                res = curr_request.get_output_tensor(0).data\n",
     "                stop_time = time.time()\n",
     "                total = stop_time - start_time\n",
     "                num = num + 1\n",
```

Comment:
Variables total and num should be more meaningful e.g. total_time, frame_number. The same for sync function